src/Core/Framework/Api/ApiDefinition/Generator/Schema/StoreApi/paths/document.json

```diff
@@ -87,6 +87,101 @@
                     }
                 ]
             }
+        },
+      "/document/download/{documentId}/{deepLinkCode}/{fileType}": {
+        "post": {
+          "tags": [
+            "Document",
+            "Endpoints supporting Criteria "
+          ],
+          "summary": "Download generated document",
+          "description": "Returns blob file of a generated document to download. Use {fileType} to select the file type (e.g. \"pdf\", \"html\").",
+          "operationId": "downloadByFileType",
+          "parameters": [
+            {
+              "name": "documentId",
+              "in": "path",
+              "required": true,
+              "schema": {
+                "type": "string",
+                "pattern": "^[0-9a-f]{32}$"
+              }
+            },
+            {
+              "name": "deepLinkCode",
+              "required": true,
+              "in": "path",
+              "schema": {
+                "type": "string"
+              }
+            },
+            {
+              "name": "fileType",
+              "required": true,
+              "in": "path",
+              "schema": {
+                "type": "string"
+              },
+              "description": "File type that selects the renderer (e.g. pdf, html)."
+            },
+            {
+              "name": "sw-language-id",
+              "in": "header",
+              "description": "Instructs Shopware to return the response in the given language.",
+              "required": false,
+              "schema": {
+                "type": "string",
+                "pattern": "^[0-9a-f]{32}$"
+              }
+            },
+            {
+              "name": "sw-include-search-info",
+              "in": "header",
+              "description": "Controls whether API search information is included in the response. Default is 1 (enabled), will be 0 (disabled) in the next major version.",
+              "required": false,
+              "schema": {
+                "type": "string",
+                "enum": [
+                  "0",
+                  "1"
+                ],
+                "default": "1"
+              }
+            }
+          ],
+          "requestBody": {
+            "required": false,
+            "content": {
+              "application/json": {
+                "schema": {
+                  "allOf": [
+                    {
+                      "$ref": "#/components/schemas/Criteria"
+                    }
+                  ]
+                }
+              }
+            }
+          },
+          "responses": {
+            "200": {
```

Comment:
should we add the `400` response in case an invalid / non supported filetype is provided?